src/pool/pool_jemalloc.c

```diff
@@ -47,6 +52,14 @@ static jemalloc_memory_pool_t *get_pool_by_arena_index(unsigned arena_ind) {
     return pool_by_arena_index[arena_ind];
 }
 
+// SplitMix64 hash
```

Comment:
source? license? 
the best would be to either use something from std or a separate header file with proper license

Reply:
i copied it from memkind so it's fine.